content/tutorial/godot/2D/normal-map-hints/index.md

```diff
@@ -0,0 +1,224 @@
+
++++
+title = "Normal Map Hints"
+description = "Lear how to use Normal Maps with Godot for 2D games."
+author = "Pablo"
+
+date = 2020-09-18T18:12:50-03:00
+weight = 5
+draft = true
+
+difficulty = "beginner"
+keywords = ["tutorial", "2D"]
```

Comment:
For keywords, think like expressions that people would search on a search engine. E.g. "Godot normal map", 2d normal map", "2d game lighting",...